
If you rename this patch, check that debian/mk/build.mk is in sync with the
new name.

--- src/sys/conf/newvers.sh.old	2004-07-17 02:07:10.000000000 +0200
+++ src/sys/conf/newvers.sh	2004-07-17 02:10:15.000000000 +0200
@@ -37,7 +37,7 @@
 TYPE="FreeBSD"
 REVISION="5.2.1"
 BRANCH="RELEASE"
-RELEASE=5.2.1-RELEASE
+RELEASE=5.2.1-@revision@
 VERSION="${TYPE} ${RELEASE}"
 
 if [ "X${PARAMFILE}" != "X" ]; then
@@ -91,7 +91,7 @@
 $COPYRIGHT
 char sccspad[32 - 4 /* sizeof(sccs) */] = { '\\0' };
 char sccs[4] = { '@', '(', '#', ')' };
-char version[] = "${VERSION} #${v}: ${t}\\n    ${u}@${h}:${d}\\n";
+char version[] = "#${v}: ${t}\\n";
 char ostype[] = "${TYPE}";
 char osrelease[] = "${RELEASE}";
 int osreldate = ${RELDATE};
